packages/react/src/box/styles.ts

```diff
@@ -466,6 +472,7 @@ export function boxStyles({
 	focus,
 	link,
 	highContrastOutline,
+	breakWords,
```

Comment:
I guess this should be moved under `textAlign` so it's in the same typography group order